code/game/objects/items/weapons/handcuffs.dm

```diff
@@ -141,6 +141,7 @@ var/last_chew = 0
 	name = "fuzzy cuffs"
 	icon_state = "fuzzycuff"
 	desc = "Use this to keep... 'prisoners' in line."
+	breakkouttime = 30 //3sec breakout time. why did this not exist before. bruh moment.
```

Comment:
Extra k's making Travis screech here.

Reply:
when the fuck did you start paying attention to this codebase